ansible/vars/enikshay/enikshay_public.yml

```diff
@@ -28,6 +28,7 @@ couchdb2:
   username: "{{ localsettings_private.COUCH_USERNAME }}"
   password: "{{ localsettings_private.COUCH_PASSWORD }}"
 
+riak_backend: "leveldb-bad-config"
```

Comment:
Well that sucks

Reply:
yeah, disappointed that this cluster got setup that way 😞